src/main/java/model/UserRequestConverter.java

```diff
@@ -1,7 +1,137 @@
 package model;
 
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Pattern;
+
 public class UserRequestConverter {
     public static UserRequest convert(String input){
-        return new UserRequest();
+    	UserRequest A = new UserRequest();
+    	
+    	String[] split = input.split(",");
+    	
+        A.setCommand(getcommand(split[0]));
+
+    	A.setArguments(getArgument(split));
+    	
+    	A.setOptions(getOption(split));	
+    	
+        return A;
+    }
+
+
+    public static boolean isValidCommandTrue(String str) {
+    	return str.equals("ADD") || str.equals("DEL") || str.equals("MOD") || str.equals("SCH");
+    }
+
+    public static boolean isValidnameTrue(String str, String option) {
+
+    	String regx_name = "^[A-Z]*\\s[A-Z]*$";
+    	if(option.equals("-f") || option.equals("-l")) regx_name = "^[A-Z]*$";
+    	return Pattern.matches(regx_name, str) && str.length() <= 15;
+    }
+    
+    public static boolean isValidclTrue(String str) {
+    	String regx_cl = "^CL[1-4]$";
+    	return Pattern.matches(regx_cl, str);
+    }
+    
+    public static boolean isValidphoneNumTrue(String str, String option) {
+    	String regx_phone = "^010-(?:\\d{3}|\\d{4})-\\d{4}$";
```

Comment:
```suggestion
    	String regx_phone = "^010-\\d{4}-\\d{4}$";
```
이렇게 변경해야 하지 않을까요? 밑에 옵션 조건보니 가운데도 항상 4개의 숫자이네요 :)